src/transformers/tokenization_utils_base.py

```diff
@@ -1929,6 +1930,9 @@ def tojson(x, ensure_ascii=False, indent=None, separators=None, sort_keys=False)
             # We also expose some options like custom indents and separators
             return json.dumps(x, ensure_ascii=ensure_ascii, indent=indent, separators=separators, sort_keys=sort_keys)
 
+        def strftime(format):
+            return datetime.now().strftime(format)
```

Comment:
It's strftime but only for the now datetime object. It would be better to do something like `def strftime_now(format)`

Reply:
Done!